src/markdown/mas/distance_sampling.Rmd

```diff
@@ -1392,6 +1180,8 @@ veldleeuwerik_df %>%
 
 # Densiteitsschattingen 2022 {#dens-2022}
 
+Later kunnen we dezelfde analyses doen voor 2023.
```

Comment:
Is het je bedoeling om dit in deze PR al toe te voegen?
Ga je deze modellen apart per jaar fitten of ga je jaar toevoegen als extra factor in de modellen?